packages/contentful--app-scripts/src/create-app-definition/build-app-definition-settings.ts

```diff
@@ -87,14 +94,61 @@ NOTE: This will create an app definition in your Contentful organization.
         return true;
       },
     },
+    {
+      name: 'pageNav',
+      message: 'Page location: Would you like your page location to render in the main navigation?',
+      type: 'confirm',
+      default: false,
+      when(answers) {
+        return answers.locations.includes('page');
+      },
+    },
+    {
+      name: 'pageNavLinkName',
+      message: 'Page location: Provide a name for the link in the main navigation:',
+      when(answers) {
+        if (answers.locations.includes('page') && answers.pageNav) {
+          return answers.name;
```

Comment:
does returning this value rather than a bool here have some special behavior? I couldn't find anything in the docs, was wondering if we might want to switch this to return a bool rather than something truthy in case inquirer gets more strict in the future

Reply:
Yes, we should have this as a bool. I think this was leftover from some trial and error around trying to utilizing the result of the name prompt as a default here (I was unsuccessful 😞). So I will change this back to a boolean